src/lsm/lsm_work_unit.c

```diff
@@ -117,17 +117,13 @@ __wt_lsm_get_chunk_to_flush(WT_SESSION_IMPL *session,
 	} else
 		chunk = (evict_chunk != NULL) ? evict_chunk : flush_chunk;
 
-	if (chunk != NULL) {
```

Comment:
Thanks, Alex, I'll go back and figure out what Coverity was really complaining about, I clearly missed something. Here it is, if you're interested: https://coverity.mongodb.com/reports.htm#v12392/p10001/fileInstanceId=6667452&defectInstanceId=2702160&mergedDefectId=47823